packages/wallet/ui/src/components/Proposal.jsx

```diff
@@ -180,6 +181,12 @@ const Proposal = ({ offer, purses }) => {
           <pre>{JSON.stringify(args, null, 2)}</pre>
         </div>
       )}
+      {error !== undefined && (
```

Comment:
why not show the error if it's ever a string or other truthy object?
```suggestion
      {error && (
```

Reply:
Good point, just copypasta, done.